apps/docs/content/components/River/react.mdx

```diff
@@ -170,6 +170,37 @@ import {River} from '@primer/react-brand'
 </River>
 ```
 
+### River breakout
+
+```jsx live
+<RiverBreakout>
+  <RiverBreakout.A11yHeading>Accelerate workflows</RiverBreakout.A11yHeading>
+  <RiverBreakout.Visual>
+    <PlaceholderImage />
```

Comment:
I think `<PlaceholderImage />` only works in Storybook, not docs because the fixture is Storybook-specific

![Screenshot 2023-08-16 at 15 04 04](https://github.com/primer/brand/assets/13340707/107f7662-9d4f-49f1-b8d9-4dee86a102b9)



```suggestion
    <img
      src="https://via.placeholder.com/600x400/f5f5f5/f5f5f5.png"
      alt="placeholder with an off-white background color"
    />
```